cal/events-at-cu.py

```diff
@@ -0,0 +1,31 @@
+import requests
+from icalendar import Calendar
+from cal.schema import db, User, Event
+
+def update_from_eventsatcu():
+
+    url = "https://events.columbia.edu/feeder/main/eventsFeed.gdo?f=y&sort=dtstart.utc:asc&format\=text/calendar&count=200"
```

Comment:
I think there's an extra `\` that's not needed: 
```
url = "https://events.columbia.edu/feeder/main/eventsFeed.gdo?f=y&sort=dtstart.utc:asc&format\=text/calendar&count=200"
                                                                                             ^
```

This is also still a little long. Maybe we should do something like:
```Python
url = "https://evens.columbia.edu/feeder/main/eventsFeed.gdo" + \
       "?f=y&sort=dtstart.utc:asc&format=text/calendar&count=200"
```

Could you also document why we need each part of the query string does? (For example, why do we need to specify `&count=200`? What's going on with the `f=y`?).